modules/jolt_physics/SCsub

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+from misc.utility.scons_hints import *
+
+Import("env")
+Import("env_modules")
+
+env_jolt = env_modules.Clone()
+
+# Thirdparty source files
+
+thirdparty_dir = "#thirdparty/jolt_physics/"
+thirdparty_sources = [
+    "Jolt/RegisterTypes.cpp",
+    "Jolt/AABBTree/AABBTreeBuilder.cpp",
+    "Jolt/Core/Color.cpp",
+    "Jolt/Core/Factory.cpp",
+    "Jolt/Core/IssueReporting.cpp",
+    "Jolt/Core/JobSystemSingleThreaded.cpp",
+    "Jolt/Core/JobSystemThreadPool.cpp",
+    "Jolt/Core/JobSystemWithBarrier.cpp",
+    "Jolt/Core/LinearCurve.cpp",
+    "Jolt/Core/Memory.cpp",
+    "Jolt/Core/Profiler.cpp",
+    "Jolt/Core/RTTI.cpp",
+    "Jolt/Core/Semaphore.cpp",
+    "Jolt/Core/StringTools.cpp",
+    "Jolt/Core/TickCounter.cpp",
+    "Jolt/Geometry/ConvexHullBuilder.cpp",
+    "Jolt/Geometry/ConvexHullBuilder2D.cpp",
+    "Jolt/Geometry/Indexify.cpp",
+    "Jolt/Geometry/OrientedBox.cpp",
+    "Jolt/Math/Vec3.cpp",
+    # "Jolt/ObjectStream/ObjectStream.cpp",
```

Comment:
Remove unused comments.

Reply:
They're there to signify that the files are deliberately excluded, so nobody else goes and adds them in the future, as that particular feature isn't used by this module.

I added a comment to better explain why they're left out.